config/development.toml

```diff
@@ -560,6 +560,7 @@ delay_between_retries_in_milliseconds = 500
 
 [kv_config]
 ttl = 900 # 15 * 60 seconds
+soft_kill = false
```

Comment:
Can you add this in `config/docker_compose.toml` as well?

Reply:
sure